### ISSUE TITLE
graph-3/status-warning colors too similar

### ISSUE THREAD

**Issue by jensy** [grommet/grommet-theme-hpe#72]
<img width="837" alt="image" src="https://user-images.githubusercontent.com/105723/83294908-5c67af80-a1ab-11ea-96f4-62d16369b158.png">

We noticed working with data viz more that `graph-3` is awfully close to `status-warning`, see screenshot.

Suggestion: Do not use yellow as a graph color at all if possible, or not any color that comes close to status colors.

**Comment by lorbritton** [comment]
I would also suggest removing orange from the graphs as that is often used as a status color.

**Comment by gregfuruiye** [comment]
In the context of our existing color palettes, I'm proposing we could replace graph-3 'yellow!' with the light purple color. In considering orange, I wonder if it could still be used within a chart in the right context.
<img width="506" alt="Screen Shot 2020-06-25 at 12 01 51 PM" src="https://user-images.githubusercontent.com/17889451/85774002-aa54d200-b6db-11ea-8de2-39b301f7e0e6.png">

**Comment by jensy** [comment]
Can we talk about this with the team  coming Wednesday @gregfuruiye?

**Comment by gregfuruiye** [comment]
We could, an important part of the entire conversation is to include how accessibility issues with charting are part of this discussion. In many ways, color is not the primary issue when it comes to accessible charting. Matt Glissman and Eric Soderberg spent a lot of time last week researching charting, focused on accessibility. Color was not the issue as opposed to what the chart is used for and what it is seeking to communicate.

**Comment by ecircenis** [comment]
While accessibility is a requirement, let's not confuse this request with accessibility.  The issue is that when color is used for non-status purposes, it should not compete with status.  E.g. it would be really bad if the color palette for graphs consisted of only reds, yellows and greens.  Why? Because this would dilute the meaning of status colors and make it harder to see, at a glance, where you might have status issues.  The ask is that graph colors be drawn from a set that are not easily confused with status colors.  Purples, Blues, and Browns come to mind immediately.  

Also, when considering color, be sure to also evaluate choices with Night Light / Night Shift enabled.

**Comment by gregfuruiye** [comment]
@ecircenis, agreed and well understood that this issue is entirely related to creating separation between graph color and status color.

**Comment by taysea** [comment]
Bumping conversation on this once more after discussion was raised with Ezmeral wanting to convey meaning via graph colors for status but finding it too similar to the normal graph colors
